project/Dependencies.scala

```diff
@@ -0,0 +1,261 @@
+    }
+
+    object TestBinaries {
+        val akkaTestkit            = Def.setting {"com.typesafe.akka"            %% "akka-testkit"             % akkaVersion.value}
+        val akkaStreamTestkit      = Def.setting {"com.typesafe.akka"            %% "akka-stream-testkit"      % akkaVersion.value}
+        val akkaHttpTestkit        = Def.setting {"com.typesafe.akka"            %% "akka-http-testkit"        % akkaHttpVersion.value}
+        val scalaTest              = "org.scalatest"                             %% "scalatest"                % "3.2.2"
+        val gatlingHighcharts      = "io.gatling.highcharts"                      % "gatling-charts-highcharts"% "3.7.5"
+        val gatlingTestFramework   = "io.gatling"                                 % "gatling-test-framework"   % "3.2.1"
+    }
+
+    import Compile._
+
+    val l = libraryDependencies
+
+    val webapiLibraryDependencies = l ++= Seq[sbt.ModuleID](
+        akkaActor.value,
+        akkaHttp.value,
+        akkaHttpCirce,
+        akkaHttpCors,
+        akkaHttpJacksonJava.value,
+        akkaHttpSprayJson.value,
+        akkaHttpXml.value,
+        akkaProtobufV3.value,
+        akkaSlf4j.value,
+        akkaStream.value,
+        apacheHttpClient,
+        bcprov,
+        chill,
+        commonsBeanUtil,
+        commonsIo,
+        commonsText,
+        commonsValidator,
+        diff,
+        ehcache,
+        gwtServlet,
+        icu4j,
+        jacksonScala,
+        jaxbApi,
+        jedis,
+        jenaLibs.value,
+        jenaText.value,
+        jodaConvert,
+        jodaTime,
+        jodd,
+        jwtSprayJson,
+        kamonCore,
+        kamonLogback,
+        kamonPrometheus,
+        kamonScalaFuture,
+        logbackClassic,
+        rdf4jClient,
+        rdf4jRuntime,
+        rdf4jStorage,
+        saxonHE,
+        scalaGraph,
+        scalaJava8Compat,
+        scalaLogging,
+        scalaXml,
+        scallop,
+        springSecurityCore,
+        swaggerAkkaHttp,
+        typesafeConfig,
+        WebapiTest.akkaHttpTestkit.value,
+        WebapiTest.akkaStreamTestkit.value,
+        WebapiTest.akkaTestkit.value,
+        WebapiTest.gatlingHighcharts,
+        WebapiTest.gatlingTestFramework,
+        WebapiTest.scalaTest,
+        WebapiTest.testcontainers,
+        xmlunitCore,
+        zio,
+        zioJson,
+        zioPrelude,
+        zioTest,
+        zioTestJunit,
+        // missing from current BAZEL setup
+        logbackCore,
+        log4jOverSlf4j,
+        jclOverSlf4j,
+        slf4jApi,
+        titaniumJSONLD,
+        jakartaJSON,
+        twirlApi,
+        junit,
+        seleniumSupport
+    )
+
+    val upgradeLibraryDependencies = l ++= Seq[sbt.ModuleID](
+        rdf4jRuntime,
+        scallop
+    )
+
+    val webapiTestAndITLibraryDependencies = l ++= Seq[sbt.ModuleID](
+        TestBinaries.gatlingHighcharts,
+        TestBinaries.gatlingTestFramework,
+        TestBinaries.scalaTest
+    )
+}
```

Comment:
please add "new line at end of file" to your VS Code settings, if you haven't done so already :)

Reply:
Good point. I think we should also share the core vscode settings.